cli/src/pcluster/config/cluster_config.py

```diff
@@ -1260,6 +1264,19 @@ def is_dcv_enabled(self):
         """Return True if DCV is enabled."""
         return self.head_node.dcv and self.head_node.dcv.enabled
 
+    @property
+    def are_all_security_groups_customized(self):
+        """Return True if all of head node and queues have (additional) security groups specified."""
```

Comment:
minor: Return True if both head node and ...

Reply:
I believe all is better because there are multiple queues